### PR TITLE
Improve Japanese translation

### PR DESCRIPTION
**Pull request by Layzie**
@makotot @sindresorhus @forresst  

Now I review & rewrite `readme.md` and `contributing.md`.

I'll rewrite `docs/recipes/` and `code-of-conduct.md` soon.

### CODE REVIEW

**Comment by forresst** [comment]
@Layzie Sorry, I'm french, for the technical part is ok.

@makotot @Rokt33r can you review this tranlations ?

**Comment by makotot** [comment]
@forresst Yes, I'll do.

**Comment by forresst** [comment]
@makotot ping :)

File: ja_JP/readme.md [hunk 1]
```diff
@@ -653,9 +621,9 @@ t.ok(x === 'bar')
      "foo"
 ```
 
-このような場合には`t.is()`を使えて、またそうすべきでしょうが、これはただの簡単な例です。
+実際はこのような場合には`t.is()`を使えますし、そうしたほうが良いですえが、これはただの簡単な例です。
```
Comment (makotot) [inline ja_JP/readme.md]: It might be `そうしたほうが良いですが`.

**Comment by makotot** [comment]
@forresst @Layzie 
Sorry for late reply. I pointed out only one typo. Except for this typo, LGTM.

**Comment by Layzie** [comment]
@makotot @forresst Thanks to review. I've fixed typo.

**Comment by makotot** [comment]
@Layzie @forresst Thanks :)

**Comment by forresst** [comment]
:+1: thanks

**Pull request merged**